backend/src/org/akvo/dash/endpoint/transformation.clj

```diff
@@ -0,0 +1,11 @@
+(ns org.akvo.dash.endpoint.transformation
+  (:require [compojure.core :refer :all]
+            [org.akvo.dash.component.tenant-manager :refer [connection]]
+            [org.akvo.dash.transformation :as t]))
+
+(defn endpoint [{:keys [tenant-manager config]}]
+  (context "/api/transformations" {:keys [tenant] :as request}
+    (let-routes [tenant-conn (connection tenant-manager tenant)]
+      (context "/:id" [id]
```

Comment:
If we only ever have one endpoint we could drop the context here and do `(POST "/:id" ...`?